api/1/point/points/get/response-example.json

```diff
@@ -116,5 +116,32 @@
     "surfaceCode": "surfaceCode",
     "timestampMs": 12345,
     "type": "OffsetLinePoint"
+  },
+  {
+    "abandoned": false,
+    "code": "code",
+    "description": "hole point description",
+    "length": 3.2,
+    "lineId": "1",
+    "localId": 6,
+    "machineId": "00000000-0000-0000-0000-111111111111",
+    "ref": {
+      "code": "refCode",
+      "dh": 1,
+      "dx": 2,
+      "dy": 3,
+      "dz": 4,
+      "filePath": "refFilePath.xml",
+      "name": "refName"
+    },
+    "remoteId": 6,
+    "sequenceId": 6,
+    "siteId": "00000000-0000-0000-0000-000000000001",
+    "status": "EARLY",
+    "timestampMs": 12345,
+    "type": "AsbuiltHolePoint",
+    "x": 123,
+    "y": 123,
+    "z": 123
```

Comment:
Näitä ehkä pitäisi olla kaksi, samalla siitä näkisi että mitkä kaikki tiedot muuttuvat ja mitkä ovat aina samat molemmilla pisteillä.